packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -228,6 +247,12 @@ export interface IFluidContainer<TContainerSchema extends ContainerSchema = Cont
 	 * Dispose of the container instance, permanently disabling it.
 	 */
 	dispose(): void;
+
+	/**
+	 * Allows the host to have the container force to be in read-only mode
+	 * @param readonly - Boolean that toggles if read-only policies will be enforced
+	 */
+	forceReadonly(readonly: boolean): void;
```

Comment:
I think it's fine to have it here, but this feels optional. It's very useful in our encapsulated model (i.e. when apps use IContainer) as we have a component developer role there, component developers need to know if they can allow editing or not). if app is using OdspClient, then app developer is exposed only to this layer, and they know (or should better know) if they would want to treat container as read-only or not.